index.d.ts

```diff
@@ -20,6 +20,7 @@ console.log(count);
 ```
 */
 export default function pDoWhilst<ValueType>(
```

Comment:
You also need to update the doc comment.

Reply:
Updated in 2f24285.